src/en/reference-constraints.md

```diff
@@ -102,7 +102,7 @@ addresses the situation.
 - Valid values: instance-type=[defined on the cloud]
 
 ### LXD:
-- Unsupported: [cpu-cores, cpu-power, instance-type, tags, virt-type]
+- Unsupported: [cpu-cores, cpu-power, tags, virt-type]
```

Comment:
You will get a warning with _cpu-cores_, as it is a deprecated alias of _cores_; but technically it is supported.

Reply:
let's go ahead and make sure to update to just cores as we go then please.

Reply:
The charms-bundles.md page contained 'cpu-cores' and was updated.